tests/utils.go

```diff
@@ -2084,12 +2084,14 @@ func GetGuestAgentUserData() string {
 	ipv6UserDataString := ""
 	if netutils.IsIPv6String(dnsServerIP) {
 		ipv6UserDataString = fmt.Sprintf(`sudo ip -6 addr add fd10:0:2::2/120 dev eth0
-                             sudo ip -6 route add default via fd10:0:2::1 src fd10:0:2::2
+                             for i in {1..20}; do sudo ip -6 route add default via fd10:0:2::1 src fd10:0:2::2 && break || sleep 0.1; done
                              echo "nameserver %s" >> /etc/resolv.conf`, dnsServerIP)
 	}
 	return fmt.Sprintf(`#!/bin/bash
                 echo "fedora" |passwd fedora --stdin
-				%s
+                systemctl disable NetworkManager
```

Comment:
Hey I think we shouldnt disable NM, if NM takes control and overwrite addresses 
we should first check if NM is installed and if it is then configure what needed though NM. 
We might miss issues we want to take care of caused by NM with kubevirt

Reply:
The need here is to have connectivity, or more accurately, to solve a problem of connectivity.
Starting to work with NM at this stage is an overkill and not needed.

The real solution is to use cloud-init network options, which will probably translate to NM ifcfg files. But this needs more work and investigation.